__tests__/api/commonFunctions.test.ts

```diff
@@ -0,0 +1,82 @@
+import {
+	rejectIfCondition,
+	resolveIfValid,
+	prepareForForwarding,
+	forwardRequest,
+	handleLogin,
+	handleRefresh
+} from "../../lib/apiFunctions/commonFunctions";
+import mock from "mock-http";
+import { NextApiRequest } from "next";
+
+describe("Test the functions that make the proxy work", () => {
+	let req: NextApiRequest,
+		res: any,
+		rej: any,
+		resolve: any,
+		token =
+			"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE5OTk5OTk5OTksImVtYWlsIjoidGVzdEB0ZXN0LmNvIiwiaWF0IjoxNTE2MjM5MDIyfQ.JyrM7CQymsHChoqFcj_-VCJwn0mDQtN9r8jnEJ_TySw";
```

Comment:
is this a valid one ?

Reply:
No, it's one I generated with jwt.io. Actually, all the tokens that you can find in the test are generated with that tool